homeassistant/components/notify/kodi.py

```diff
@@ -0,0 +1,73 @@
+﻿"""
+Kodi notification service.
+
+For more details about this platform, please refer to the documentation at
+https://home-assistant.io/components/notify.kodi/
+"""
+import logging
+
+from homeassistant.components.notify import (ATTR_TITLE, ATTR_TITLE_DEFAULT,
+                                             ATTR_DATA,
+                                             BaseNotificationService)
+
+_LOGGER = logging.getLogger(__name__)
+REQUIREMENTS = ['jsonrpc-requests==0.3']
+
+ATTR_DISPLAYTIME = 'displaytime'
+ATTR_ICON = 'icon'
+
+
+def get_service(hass, config):
+    """Return the notify service."""
+    url = '{}:{}'.format(config.get('host'), config.get('port', '8080'))
+
+    jsonrpc_url = config.get('url')  # deprecated

```

Comment:
For a new component you can remove this deprecated lines. Simply use `url` from line 22 as the correct URL